tests/model/test_model.py

```diff
@@ -3346,6 +3347,46 @@ def test__handle_subscription_event_subscribers_one_user_multiple_streams(
             new_subscribers = model.stream_dict[stream_id]["subscribers"]
             assert new_subscribers == expected_subscribers
 
+    @pytest.mark.parametrize(
+        "person, changed_details",
+        [
+            (
+                {"user_id": 11, "full_name": "New Full Name"},
+                "full_name",
+            ),
+            ({"user_id": 11, "timezone": "New Timezone"}, "timezone"),
+            ({"user_id": 11, "is_billing_admin": False}, "is_billing_admin"),
+            ({"user_id": 11, "role": 10}, "role"),
+            (
+                {"user_id": 11, "avatar_url": "new_avatar_url", "avatar_version": 21},
+                "avatar_url",
+            ),
+            ({"user_id": 11, "new_email": "new_display@email.com"}, "email"),
+            (
+                {"user_id": 11, "delivery_email": "new_delivery@email.com"},
+                "delivery_email",
+            ),
+        ],
+        ids=[
+            "full_name",
+            "timezone",
+            "billing_admin_role",
+            "role",
+            "avatar",
+            "display_email",
+            "delivery_email",
+        ],
+    )
+    def test__handle_realm_user_event(
+        self, person, changed_details, model, initial_data
+    ):
+        event = {"type": "realm_user", "op": "update", "id": 1000, "person": person}
+        model._handle_realm_user_event(event)
+        assert (
+            initial_data["realm_users"][1][changed_details]
```

Comment:
I'm guessing the `1` is the index into initial_data that corresponds to `user_id=11`?

If that's the case, it would be more reliable to make that explicit in the test code, eg. if the initial data gets reordered at a later stage, this would make this test fail even though it's functionally correct?

Reply:
Okay, I'll add a comment for this 👍